harpy/assembly.py

```
@@ -11,90 +12,133 @@
     os.makedirs(workflowdir, exist_ok=True)
     fetch_rule(workflowdir, f"{asm}.smk")
     os.makedirs(f"{output_dir}/logs/snakemake", exist_ok = True)
     sm_log = snakemake_log(output_dir, asm)
+    configs = {
+        "workflow" : asm,
+        "snakemake_log" : sm_log,
+        "output_directory" : output_dir,
+        "barcode_tag" : bx_tag.upper(),
+        "spades" : {
+            **({'assembler' : metassembly} if metassembly else {}),
+            "k" : 'auto' if kmer_length is "auto" else ",".join(map(str,kmer_length)),
```

Ruff

Use `==` to compare constant literals

Replace `is` with `==`

(F632)